src/tribler-core/tribler_core/modules/resource_monitor.py

```diff
@@ -167,9 +167,8 @@ def write_resource_logs(self, time_seconds):
         if not self.memory_data or not self.cpu_data:
             return
         with self.resource_log_file.open(mode="a+") as output_file:
-            output_file.write(u"%s, %s, %s\n" % (time_seconds,
-                                                self.memory_data[len(self.memory_data)-1][1],
-                                                self.cpu_data[len(self.cpu_data)-1][1]))
+            output_file.write(f"{time_seconds}, {self.memory_data[len(self.memory_data)-1][1]}, "
```

Comment:
Apparently, this change [breaks](https://jenkins-ci.tribler.org/job/GH_Tribler_PR_Tests/job/PR_async_win64/205/artifact/output/test_runners_output/3.err/*view*/) the resources log test on Windows:   
```

======================================================================
FAIL: Test resource log file is created when enabled.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\users\heteoma\appdata\local\programs\python\python37\lib\site-packages\asynctest\case.py", line 297, in run
    self._run_test_method(testMethod)
  File "c:\users\heteoma\appdata\local\programs\python\python37\lib\site-packages\asynctest\case.py", line 352, in _run_test_method
    result = method()
  File "C:\workspace\jenkins2\workspace\GH_Tribler_PR_Tests\PR_async_win64\tribler\src\tribler-core\tribler_core\tests\tools\test_as_server.py", line 54, in check
    result = fun(*argv, **kwargs)
  File "C:\workspace\jenkins2\workspace\GH_Tribler_PR_Tests\PR_async_win64\tribler\src\tribler-core\tribler_core\modules\tests\test_resource_monitor.py", line 110, in test_resource_log
    self.assertTrue(self.resource_monitor.resource_log_file.exists())
AssertionError: False is not true
-------------------- >> begin captured logging << --------------------
DEBUG   1579336343 selector_events:asyncio:53   Using selector: SelectSelector
DEBUG   1579336343 resource_monitor:ResourceMonitor:110  Checking memory/CPU usage
ERROR   1579336343 resource_monitor:ResourceMonitor:138  Failed to get memory full info: psutil.AccessDenied (pid=4828)
--------------------- >> end captured logging << ---------------------
```

Unfortunately, some tunnel-related tests on Windows do not pass yet as a result of our big refactorings. This masks other tests fail, so one should pay more attention to these. Also, the fact that our Windows testers do not provied an aggregate test results view does not help either.

Reply:
I wonder what that is… Oh, well. Time to revert the change

Reply:
Weird that this of all things would break code. Nonetheless the change is reverted (except removing the superfluous u-prefix from the string).